docs/installation/production/additional/official_docker.rst

```diff
@@ -0,0 +1,423 @@
+.. _`Salt Script`: https://docs.saltstack.com/en/latest/topics/index.html
+.. _production_official_docker:
+
+*********************
+Official Docker Image
+*********************
+
+**Last Updated:** July 2020
+
+Versions and Tags
+#################
+
+The official Tethys Docker images are located in the `tethysplatform/tethys-core Docker Hub repository<https://hub.docker.com/r/tethysplatform/tethys-core>`_. Images released are tagged using the following format:
+
++---------------+------------------------------------------------------------------------------------------------------+
+|    Tag        | Description                                                                                          |
++===============+======================================================================================================+
+| latest        | latest tagged released                                                                               |
++---------------+------------------------------------------------------------------------------------------------------+
+| master        | latest development build (what's in the master branch)                                               |
++---------------+------------------------------------------------------------------------------------------------------+
+| version       | Specify the version of Tethys image, for example: v3.0.5                                             |
++---------------+------------------------------------------------------------------------------------------------------+
+
+Pull a Pre-Built Image
+######################
+
+The following commands will download Tethys image to your machine:
+
+::
+
+    docker pull tethysplatform/tethys-core:latest           # the latest Tethys release
+
+::
+
+    docker pull tethysplatform/tethys-core:v3.0.5           # Tethys version 3.0.5
+
+::
+
+    docker pull tethysplatform/tethys-core:master           # the latest development build
+
+Build From Source
+#################
+
+Make sure that there isn't already a docker container or docker images with the desired name:
+
+::
+
+    docker rm tethys-core
+    docker rmi tethys-core
+
+To build a new docker image with the desired name and tag, run the following in the directory containing the Dockerfile:
+
+::
+
+    docker build -t tethys-core:latest
+
+Run Image
+#########
+
+When running the docker you can use the ``-e`` flag to set environment variables. A list of available environment variables is given in a table below.
+
+::
+
+    -e TETHYS_CONDA_ENV='tethys'
+
+.. note::
+
+    Variables in the build arg table can be used here as well.
+
+
+Example of Run Command:
+
+::
+
+    docker run -p 127.0.0.1:80:80 --name tethys-core \
```

Comment:
Does this example run command work?

Reply:
I tested it and it works. I need to update the command to use tethysplatform/tethys-core instead of tethys-core.